src/components/EventDetails/EventDetails.js

```diff
@@ -431,14 +431,15 @@ export class EventDetail extends React.Component<Props> {
     return isReceived ? event.from : event.to;
   };
 
-  getFormattedGasFee = (formattedFee: number, token: string) => {
+  getFormattedGasFee = (formattedFee: number) => {
     const { baseFiatCurrency, ratesPerChain } = this.props;
     const ethereumRates = ratesPerChain[CHAIN.ETHEREUM] ?? {};
     const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
-    const rate = getRate(ethereumRates, token, fiatCurrency);
+    const { symbol, address } = nativeAssetPerChain.ethereum;
+    const rate = getAssetRateInFiat(ethereumRates, address, fiatCurrency);
```

Comment:
Potentially rate could be nil

Reply:
```
export const getAssetRateInFiat = (
  rates: RatesByAssetAddress,
  assetAddress: string,
  fiatCurrency: Currency,
): number => valueForAddress(rates, assetAddress)?.[fiatCurrency] ?? 0;
```

So might be just 0, not nil, unless I miss something? 🤔

Reply:
I would differentiate between zero value and missing value for the assets